addons/survey/models/survey_survey.py

```diff
@@ -1258,6 +1258,34 @@ def _create_certification_badge_trigger(self):
             'target_goal': 1
         })
 
+    def _get_languages(self):
+        """ Return languages available for the survey ((code, name) pairs sorted by name).
+
+        A language is considered to be available for the survey
+        if the language is available in the front-end AND:
+        - if the survey title is translated or
+        - if at least one question/page title is translated in that language
+        Notes:
+        - "translated" means that the title value is different in that language than in the base language ('en_US').
+        - if the base language is present in the frontend languages, it is always returned
+        - otherwise it is never returned and
+          the result can be empty if the translations in all other language are equal to the ones in the base language.
+        """
+        self.ensure_one()
+        langs = self.env['res.lang']._get_frontend()
+        lang_base = 'en_US'
```

Comment:
I think you can use the first language from `langs` instead, no? 
If all translations are equal to this, we'll show no option, if some differ, this first lang will be a valid option.
The hardcoded "en_US" part is an implementation of the ORM for the db record, we don't need to explicitly use it.

Reply:
Yes, maybe, it would be easier to get rid of this auto-detection or accept it may make some mistakes as I haven't found a way by not using SQL nor en_US as base language to "We trust those frontend languages (but we avoid showing selector if nothing was translated)". As if 2 languages have the same translation, we cannot know which one must be included or even if none of them must be.
What do you think ? Do you have a better idea ?

Reply:
Yes :) That, or explicitly setting available translations on the survey, following PO's preference.